File: Python PowerPoint Examples/PlaceholdersApiDemo/getSlidesPlaceholder.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.placeholders_api.PlaceholdersApi(configuration)

fileName = "getSlidesPlaceholder.pptx"
password = ""
storage = ""
slideIndex = 0
inputFolder = "input"
placeholderIndex = 0
result = api.get_slides_placeholder(fileName, slideIndex, placeholderIndex,password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/PlaceholdersApiDemo/getSlidesPlaceholder.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\PlaceholdersApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new PlaceholdersApi($configuration);

$name = "GetSlidesPlaceholder.pptx";
$password = null;
$folder = "Powerpoint";
$storage = null;
$slideIndex = 0;
$placeholderIndex = 0;
$result = $apiInstance->getSlidesPlaceholder($name,$slideIndex,$placeholderIndex, $password, $folder, $storage);
print_r($result);